docs/src/pages/guides/FHIROperationFramework.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: default
+title: IBM FHIR Server - Extended Operations Framework
+date:  2020-09-29
+---
+
+# IBM FHIR Server - Extended Operations Framework
+
+The HL7 FHIR Specification describes a [light weight framework](https://www.hl7.org/fhir/r4/operations.html) of extended operations for the REST API.
+
+This framework defines REST endpoints at the Base Server (System), scoped to specific resources (Type) and scoped to specific instances. 
+- **System** - https://my-server.com/v4/$convert - Convert from one format to another
+- **Type** - https://my-server.com/v4/Patient/$export - Retrieve all the data from the Patient compartment
+- **Instance** - https://my-server.com/v4/Patient/1-2-3-4/$export - Retrieve a specific Patient data from the Patient compartment data
```

Comment:
wondering if these should have our default base path in there.  e.g. `https://my-server.com/fhir-server/api/v4`